PatternPal/PatternPal.Core/Builders/MethodCheckBuilder.cs

```diff
@@ -15,5 +15,5 @@ internal class MethodCheckBuilder : CheckCollectionBuilder
 
     public override ICheck Build() => _check;
 
-    internal Func< INode > Result => () => _check.MatchedEntity!;
+    internal Func< List<INode> > Result => () => _check.MatchedEntities!;
```

Comment:
```suggestion
    internal Func< List<INode> > Result => () => _check.MatchedEntities;
```
This is no longer requires, as `_check.MatchedEntities` is not nullable.